src/it/scala/org/ergoplatform/it/Docker.scala

```diff
@@ -77,7 +77,11 @@ class Docker(suiteConfig: Config = ConfigFactory.empty,
     val networkPort = settings.scorexSettings.network.bindAddress.getPort
 
     val containerConfig: ContainerConfig = buildContainerConfig(nodeConfig, ip, restApiPort, networkPort)
-    val containerId = client.createContainer(containerConfig, configuredNodeName + "-" + hashString).id()
+    val containerId = Try {
+      client.createContainer(containerConfig, configuredNodeName + "-" + hashString)
+    }.recoverWith { case e: ImageNotFoundException =>
+      Failure(new Exception(s"Error: docker image is missing ($e)\nRun 'sbt it:test' to generate it."))
```

Comment:
maybe it worth nesting ImageNotFoundException to the newly created one?

Reply:
It does not look nice, but we blow up in the next line anyway. It supposed to be a "help" message to just give a hint on how to fix it. Original exception is shown in log via string interpolation. However, I forgot to pass through other exception types. Fix is on the way.